pkg/utils/k8s.go

```diff
@@ -106,3 +107,10 @@ func UpdateNamespace(ctx context.Context, kubeClient client.Client, name string,
 	}
 	return kubeClient.Update(ctx, &namespace)
 }
+
+// GetServiceAccountSubjectInToken extract ServiceAccount subject from token
+func GetServiceAccountSubjectInToken(token string) string {
```

Comment:
Nit: How about moving extracting logic to `jwt.go` and calling it in this function? Extracting subject from JWTs can be used in non-kubernetes usecases too.

```go
func GetTokenSubject(token string) string { ... }

func GetSubjectFromConfig(cfg rest.Config) string {
    return GetTokenSubject(cfg.BearerToken)
}
```

Reply:
Nice. Edited now.